README.md

```diff
@@ -220,6 +220,13 @@ dayjs().hour();
 // => 19
 ```
 
+| Library | Iteration | Time       |
```

Comment:
I think the perf table should be per method.

Reply:
I agree, maybe we can put the perf results at the end of every method example, to make more sense?

Reply:
lol, just had a second look, it's already at the end of each method. 😅 
I'm not sure if the table will look too big, let's see how it goes.

Reply:
Oh true. Maybe we don't need to mention Iteration in each method then :)